mbs_results/utilities/constrains.py

```diff
@@ -237,7 +253,8 @@ def derive_questions(
     pd.DataFrame
         Original dataframe with constrains.
     """
-    derive_map = create_derive_map(df, spp_form_id)
+    derive_map = create_derive_map(df, spp_form_id)[0]
```

Comment:
Just a suggestion, either way obviously works.
```suggestion
    derive_map, _ = create_derive_map(df, spp_form_id)
```

Reply:
This also needs to be done in line 501 for  `update_derived_weight_and_winsorised_value`